yourgov/components/FormMobileFoodVendorPermit/FormTask1Container.tsx

```diff
@@ -50,9 +52,10 @@ export function FormTask1Container({
 			<Column columnSpan={{ xs: 12, md: 4, lg: 3 }}>
 				<ContentBleed visible={{ md: false }}>
 					<ProgressIndicator
+						activePath={`${pathname}?type=${typeSearchParm}`}
```

Comment:
`typeSearchParam`?

Reply:
Yeah, it's currently built weirdly and passes which selection you made on the very first screen through query params instead of storing it in session storage. I change it in Dan's PR

Reply:
Ok, awesome.